### PR TITLE
Clone options for multiple string terms in Query#term

### LINKED ISSUE #354: Query#term called with tokens array and options only last token is used
When initializing query with the output of `lunr.tokenizer` the search is performed only for the last token.

[Fiddle](https://jsfiddle.net/ndv66e5r/15/)

The issue is in [lunr.Query.prototype.term](https://github.com/olivernn/lunr.js/blob/master/lib/query.js#L178) where the same options object is reused for each clause.

### PR DESCRIPTION
**Pull request by mike1808**
Fix #354

### CODE REVIEW

File: test/query_test.js [hunk 1]
```diff
@@ -49,6 +49,17 @@ suite('lunr.Query', function () {
       })
     })
 
+    suite('multiple string terms with options', function () {
```
Comment (mike1808) [inline test/query_test.js]: this is a regression test

**Comment by olivernn** [comment]
Nice catch, thanks for the PR. All looks good, please remove the `./lunr.js` file from the commit, it will be generated when I do a release. Note this may mean that the travis tests fail, thats ok.

One day I will get time to make the build process a little less wonky.

**Comment by mike1808** [comment]
@olivernn I removed `./lunr.js` changes from the commit.

**Comment by olivernn** [comment]
Thanks, I'll put together a new release with this (and a few other things) in the next couple of days.

**Pull request merged**